test/multicall/Multicall.js

```diff
@@ -324,16 +324,16 @@ describe('Multicall Test Suite', function () {
       expect(receipt.status).to.eq(1);
     });
 
-    it('should NOT be able to aggregate 150 calls to processLongInputTx', async function () {
-      const n = 150;
+    it('should NOT be able to aggregate 200 calls to processLongInputTx', async function () {
```

Comment:
this change is needed, because of this [PR](https://github.com/hashgraph/hedera-json-rpc-relay/pull/1978) in the relay, which changes chunk size to around 50kb

Reply:
Oh is that what it is? No wonder it kept failing in the ci